
Header header

uint32 target_id 
bool delay
Vertex vertex
